modules/ROOT/pages/global-elements.adoc

```diff
@@ -0,0 +1,65 @@
+= Global Elements
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, server, components, connectors, elements, palette, global elements, configuration elements
+
+A Global Element is a reusable object containing parameters that any number of elements in a flow can share. You reference a global element from Anypoint Connectors or components in your Mule application.
+
+Use the global element to apply configuration details to multiple local elements in flows. Create one global element that defines parameters and configuration details, and then reference the global element from any flow element that uses this configuration. This practice enables you to ensure consistency across flow elements.
+
+Note that some components require their properties stored in a global element.
+
+== Prerequisites
+
+Review the features of the xref:studio::index.adoc[Anypoint Studio Visual Editor] to learn more about developing Mule applications using the Anypoint Studio graphical user interface and to understand the context in which you use global elements.
+
+[[advantages]]
+== Benefits
+
+Global elements provide many benefits to you as a Mule application developer:
+
+* Reusability
++
+You can configure a global element once, and then share that configuration among processors of the same type. Configuring global elements is particularly useful when you have to define the connection details and login credentials for an external source, such as a SaaS application or database.
+* Efficiency
++
+You don't have to change the same configuration multiple times for different processors in the flow if they all share a global element. For example, if you have multiple connectors that connect to the same service using one set of credentials, you can change the configuration in the global element, and the changes apply to all flow elements that reference the global element.
+* Flexibility
++
+An element in a flow can apply a global element’s configuration exactly as defined, or it can apply modifications to the global configuration properties.
+
+== Create a Global Element
+
+You can create global elements in two ways:
+
+* From the *Global Elements* tab in the Anypoint Studio canvas
+* From the properties panel of any Connector or Module that supports global elements
```

Comment:
```suggestion
* From the properties panel of any connector or module that supports global elements
```
Capitalize the names of connectors and modules, but not "connector" when used alone, and not "module" ever.